tardis/model/base.py

```diff
@@ -0,0 +1,7 @@
+class Radial1DModel(object):
+    def __init__(self, plasma, atom_data, structure):
```

Comment:
okay the init is going to have `velocity`, `density`, `abundance`, `t_rad`, `w`(optional defaults to None), `v_boundary_inner`, `v_boundary_outer`.